onnxruntime/core/providers/cuda/rnn/cudnn_rnn_base.h

```diff
@@ -42,7 +42,7 @@ class CudnnRNN {
     if (!cudnn_rnn_desc_)
       CUDNN_RETURN_IF_ERROR(cudnnCreateRNNDescriptor(&cudnn_rnn_desc_));
 
-    CUDNN_RETURN_IF_ERROR(cudnnSetRNNDescriptor(cudnnHandle,
+    CUDNN_RETURN_IF_ERROR(cudnnSetRNNDescriptor_v6(cudnnHandle,
```

Comment:
>cudnnSetRNNDescriptor_v6 [](start = 26, length = 24)

should we still support older cudnn v7 here?

Reply:
No. The deprecation policy is changed in cuDNN 8. Here are the details: https://docs.nvidia.com/deeplearning/sdk/cudnn-developer-guide/index.html#backward-compatibility